CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Original pdf file is deleted when using share (<https://github.com/openvinotoolkit/cvat/pull/3967>)
 - Order in an annotation file(<https://github.com/openvinotoolkit/cvat/pull/4087>)
 - Fixed task data upload progressbar (<https://github.com/openvinotoolkit/cvat/pull/4134>)
+- Email in org invitations is case sensitive (<https://github.com/openvinotoolkit/cvat/pull/4153>)
```

Comment:
I am not sure we need to add it here because IAM is the same release like this change. Literally this bug was not released yet and it's why we probably should not list the fix here.

Reply:
We did already alpha release. It isn't mentioned here. But in general I agree.